content/efcore/getting-started/tutorial-database-reference.md

```diff
@@ -1,4 +1,5 @@
-# Database Tables 
+# Database Reference
+Below are the tables and data that these tutorials work with.
```

Comment:
Could you also update the description in the models page to say something similar?